src/pages/references/authoring/editor/classes/Editor.md

```diff
@@ -119,9 +114,8 @@ This local client will act as having unsaved changes until the upload has finish
 
 | Name | Type | Description |
 | :------ | :------ | :------ |
-| `bitmapData` | [`BitmapImage`](../interfaces/BitmapImage.md) | BitmapImage resource (e.g. returned from loadBitmapImage()). |
-| `options` | `Object` | Additional configuration: |
-| `options.initialSize?` | [`RectangleGeometry`](../interfaces/RectangleGeometry.md) | Size the image is displayed at. Must have the same aspect ratio as bitmapData. Defaults to the        size the image would be created at by a UI drag-drop gesture (typically the image's full size, but scaled down        if needed to stay below an application-defined size cap). |
+| `bitmapData` | [`BitmapImage`](../interfaces/BitmapImage.md) | BitmapImage resource (e.g. returned from `loadBitmapImage()`). |
+| `options` | `Object` | Additional configuration:<br/><br/> `options.initialSize?:` [`RectangleGeometry`](../interfaces/RectangleGeometry.md)<br/><br/> Size the image is displayed at. Must have the same aspect ratio as `bitmapData`. Defaults to the size the image would be created at by a UI drag-drop gesture (typically the image's full size, but scaled down    if needed to stay below an application-defined size cap). |
```

Comment:
I think the previous multi-row formatting here was a bit easier to read (with one row per field of the `options` object), and more like what I've seen output from other JsDoc/TsDoc generators. Would prefer we switch back to that, if we're going to be manually correcting this section either way.

Reply:
Long term though, I wonder if we should just bite the bullet and create a named interface for these options. It's a bit more verbose, and means more jumping around through different docs pages to find info on all the arguments to this method, but it will at least mean that the auto-generated docs have the correct formatting right out of the box without need for repeatedly manually fixing them up...